csrankings.csv

```diff
@@ -6455,6 +6456,7 @@ Koiti Hasida,University of Tokyo,http://www.i.u-tokyo.ac.jp/edu/course/ice/membe
 Kok-Ming Leung,New York University,http://engineering.nyu.edu/people/kok-ming-leung/,NOSCHOLARPAGE
 Kolin Paul,IIT Delhi,http://www.cse.iitd.ernet.in/~kolin/,6TWEO3IAAAAJ
 Konrad Iwanicki,University of Warsaw,http://www.mimuw.edu.pl/~iwanicki/,ALTjvpMAAAAJ
+Konrad P. Körding,University of Pennsylvania,http://koerding.com/,MiFqJGcAAAAJ

```

Comment:
Please point to a page indicating this faculty member has adjunct status in a CS department (and thus can solely advise PhD students in CS). Thanks.